spec/react_on_rails/utils_spec.rb

```
@@ -100,88 +195,128 @@
         end
       end
 
-      describe ".server_bundle_js_file_path with Shakapacker enabled" do
-        before do
-          allow(Rails).to receive(:root).and_return(Pathname.new("."))
-          allow(ReactOnRails::PackerUtils).to receive(:using_packer?).and_return(true)
-          allow(ReactOnRails::PackerUtils).to receive_message_chain("packer.config.public_output_path")
-            .and_return(Pathname.new("public/webpack/development"))
-        end
+      ["shakapacker", "webpacker"].each do |packer_type|
```

rubocop (1.69.1)

[convention] Use `%w` or `%W` for an array of words.

(Style/WordArray)